components/alttextify/alttextify.app.mjs

```diff
@@ -1,11 +1,154 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "alttextify",
-  propDefinitions: {},
+  propDefinitions: {
+    async: {
+      type: "boolean",
+      label: "Async",
+      description: "whether to add the image in the background or immediately (synchronously). If async is set to true, the API response will always be successful with an empty response body.",
+      default: false,
+    },
+    lang: {
+      type: "string",
+      label: "Language",
+      description: "The language for the alt text. Supported language codes are accepted. If not provided, the account's default language is used.",
+      default: "en",
+    },
+    maxChars: {
+      type: "integer",
+      label: "Max Characters",
+      description: "Maximum length of the generated alt text.",
+    },
+    assetId: {
+      type: "string",
+      label: "Asset ID",
+      description: "The unique identifier for the asset.",
+      optional: true,
+    },
```

Comment:
`assetId` should use async options. https://apidoc.alttextify.net/#api-Image-GetImages